workers/task-requirements.txt

```diff
@@ -1,9 +1,9 @@
-requests>=2.26,<3.0
-docker>=6.0.1,<7.0
-psutil>=5.8.0,<6.0
-humanfriendly>=9.2,<10.0
-PyJWT>=2.4.0,<3.0
+requests==2.31.0
+docker==7.0.0
+psutil==5.9.8
+humanfriendly==10.0
+PyJWT==2.8.0
 kiwixstorage==0.6
-ujson>=5.1.0,<5.2
-zimscraperlib>=1.4.1,<1.5
+ujson==5.9.0
+zimscraperlib==3.3.0
```

Comment:
Given how limited the scraperlib usage in task worker, might be worth looking into replacing it with pylibzim here.

Reply:
Done in https://github.com/openzim/zimfarm/pull/929/commits/ff103262721c44561e7204e4114a5ccd0a6c3184 

I had to duplicate some code from scraperlib but it was pretty limited, I do not expect this code to change a lot in the future and it is probably worth it to depend only on pylibzim instead of the whole scraperlib and all its dependencies.

It also helped to realize that this get_zim_info code was in fact broken. I did not realized it because `zimcheck` were not activated on the `dev` setup. It is activated now.